meson.build

```diff
@@ -13,6 +13,7 @@ icondir = join_paths(datadir, 'icons', 'hicolor')
 vapidir = meson.current_source_dir() / 'vapi/'
 
 add_global_arguments('-DGETTEXT_PACKAGE="@0@"'.format(meson.project_name()), language: 'c')
+add_global_arguments(['-w'], language: 'c')
```

Comment:
Is this change intended? (i.e. not a temporary change for your own debug purposes)